### PR TITLE
Chore merge from c5f46b0 to 6d24ccc

### PR DESCRIPTION
**Pull request by julien-deramond**
- [x] https://github.com/twbs/bootstrap/commit/c5f46b020c797ff72b36b7be098310156cbf0aac
- [x] ~~https://github.com/twbs/bootstrap/commit/9cc85fd69b35a6df4b5e49e806658be3f828f07e~~: nothing to do since we don't have these vars in Boosted
- [x] https://github.com/twbs/bootstrap/commit/0d9715e0636cd79527ece5a364a74dc3d3e99923
- [x] https://github.com/twbs/bootstrap/commit/6bb60152b688cf2765e229d990c896a014c885e3
  - See https://github.com/twbs/bootstrap/pull/38948 upstream
- [x] https://github.com/twbs/bootstrap/commit/785043c06bb45692f2a55e206833066677766cf3
- [x] https://github.com/twbs/bootstrap/commit/1a82228043ee7c9a1227e01f546d55f115e1a422
- [x] https://github.com/twbs/bootstrap/commit/b5772d4ec27149bb44ed4d4a45667d3452ab3ff9
- [x] https://github.com/twbs/bootstrap/commit/ad724cb136faae7e81c233d662f3f9bf6b800fde
- [x] https://github.com/twbs/bootstrap/commit/96b0fd67f5c400431255a5f3b207042b93452235
- [x] https://github.com/twbs/bootstrap/commit/9129e4a0736772817b433950613532d29da7e92f
- [x] https://github.com/twbs/bootstrap/commit/b7e351b445908c82d7a0642960a8ceeaba1c5db4
- [x] https://github.com/twbs/bootstrap/commit/a4918e33b80684386fce1ec64fc0204e0bac6b90
- [x] https://github.com/twbs/bootstrap/commit/beca2a6c7f6bc88b6449339fc76edcda832c59e5
- [x] https://github.com/twbs/bootstrap/commit/f3dbbc9708ef490eaa40fd6cb246b501ed854031
- [x] https://github.com/twbs/bootstrap/commit/073018b09c465f14157399e1309298963cfbe7c5
- [x] ~~https://github.com/twbs/bootstrap/commit/7a4226875d23f78ca52b2c604a9d2073f3abbba4~~: can't regerenate `package-lock.json` now since some packages will be updated later on in this PR
- [x] https://github.com/twbs/bootstrap/commit/6e49236541b65b65752cf8547b791c8dff44ed3e: this one is a bit tricky cause I kept violet color to define the custom tooltips and popovers, but I've added `--bd-primary-*` Boosted-specific to handle the various container colors in the documentation to be orange and not violet.
- [x] ~~https://github.com/twbs/bootstrap/commit/a70852fb02ab4f5971272716c541f860706cd10c~~: this example doesn't exist in Boosted
- [x] https://github.com/twbs/bootstrap/commit/c5d653bcb4049d8a952b590413eb37da253ec8ee
- [x] https://github.com/twbs/bootstrap/commit/a57939a077ebaa7bbb1f14b3d8ff4bc6b44b289d
- [x] https://github.com/twbs/bootstrap/commit/e95a67edbb344219d0b74c7b28632b54600469de
- [x] https://github.com/twbs/bootstrap/commit/5205d894ba32846ef9d48cde38f46be6d055eabd
- [x] https://github.com/twbs/bootstrap/commit/36293754fffb8b4b8457d7a163eb8c8b7a8970d4
- [x] https://github.com/twbs/bootstrap/commit/bb92ec707aa607fa4a43f49e75d69ebd67132e2e
- [x] https://github.com/twbs/bootstrap/commit/f442d9c669603ac04449ae7a6e1df874c3830bce
- [x] https://github.com/twbs/bootstrap/commit/3e3be50630f1fa2052ced3afbd3a96626f5159c6
- [x] https://github.com/twbs/bootstrap/commit/3625af375821276be4d69b496ff798f19608b782
- [x] https://github.com/twbs/bootstrap/commit/d6a55b719c81ba7ea296f793768d886d13610314
- [x] https://github.com/twbs/bootstrap/commit/6d24ccc9ccfd9b3ee8253c09b63102873ed29952

### CODE REVIEW

File: site/content/docs/5.3/components/spinners.md
Comment (louismaximepiton) [inline site/content/docs/5.3/components/spinners.md:1]: Missing some changes in here https://github.com/twbs/bootstrap/pull/38632/files. Some `role="status"` are used on `aria-hidden="true"`.

File: site/content/docs/5.3/components/collapse.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Please note that while the example below has a `min-height` set to avoid excessi
 {{< /callout >}}
 
 {{< example >}}
-<p>
+<p class="d-inline-flex gap-1">
```
Comment (louismaximepiton) [inline site/content/docs/5.3/components/collapse.md:58]: Not on the right example. I think it's the one below

File: site/assets/scss/_component-examples.scss [hunk 2]
```diff
@@ -220,17 +220,18 @@
 
 // scss-docs-start custom-tooltip
 .custom-tooltip {
-  --bs-tooltip-bg: var(--bs-primary);
+  --bs-tooltip-bg: var(--bd-violet-bg);
```
Comment (louismaximepiton) [inline site/assets/scss/_component-examples.scss:223]: **[Bootstrap side]** It feels weird to have a reference to `bd-` inside our doc.

File: site/assets/scss/_component-examples.scss [hunk 3]
```diff
@@ -220,17 +220,18 @@
...
+  --bs-tooltip-color: var(--bs-white);
 }
 // scss-docs-end custom-tooltip
 
 // scss-docs-start custom-popovers
 .custom-popover {
   --bs-popover-max-width: 12.5rem;
-  --bs-popover-border-color: var(--bs-primary);
+  --bs-popover-border-color: var(--bd-violet-bg);
+  --bs-popover-header-bg: var(--bd-violet-bg);
```
Comment (louismaximepiton) [inline site/assets/scss/_component-examples.scss:232]: **[Bootstrap side]** It feels weird to have a reference to `bd-` inside our doc.

File: site/content/docs/5.3/migration.md
Comment (louismaximepiton) [inline site/content/docs/5.3/migration.md:1]: Should we add some diff to help people visualize what changed ?

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
Feels kind of weird to have custom tooltips/popovers with a violet bg even if it underlines the fact that we shouldn't use them. But I'm fine with it anyway.

Few comments to tak into account before merging!

File: site/content/docs/5.3/migration.md
Comment (julien-deramond) [reply site/content/docs/5.3/migration.md:1]: For spinners: https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2159/commits/1feae69e99f7dce7f3374ef6e4bdf755f1f33b42

Comment (julien-deramond) [reply site/content/docs/5.3/migration.md:1]: For the second one, I'm not sure since users should read the callout which has also changed.

Comment (louismaximepiton) [reply site/content/docs/5.3/migration.md:1]: Ok fine with it 👌

**Review by louismaximepiton — APPROVED** [review]
LGTM 🚀

**Pull request merged**